lib/plugins/model.js

```diff
@@ -204,13 +206,14 @@ function findModelDefinitions(rootDir, sources) {
   return registry;
 }
 
-function loadModelDefinition(rootDir, jsonFile, allFiles) {
+function loadModelDefinition(rootDir, jsonFile, allFiles, scriptExtensions) {
   var definition = require(jsonFile);
   var basename = path.basename(jsonFile, path.extname(jsonFile));
   definition.name = definition.name || _.upperFirst(_.camelCase(basename));
 
   // find a matching file with a supported extension like `.js` or `.coffee`
-  var sourceFile = fixFileExtension(jsonFile, allFiles, true);
+  var sourceFile = fixFileExtension(jsonFile, allFiles, true, scriptExtensions);
```

Comment:
The third param `onlyScriptsExportingFunction` cause `.js` files of models unloaded. 

Line 142 in utils.js checking `require.extensions['.js'] === function` which will be false when using jest.